samples/linecounter/LineCounter.csproj

```diff
@@ -1,18 +1,20 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
Could we update to `.net8.0`? It releases in couple days . That way, the additional metrics from .NET8.0 are also added.

Reply:
sure, there are a few things that need to be updated. I'd prefer to do it in a different PR, created https://github.com/Azure/azure-sdk-for-net/issues/39960